.github/workflows/macos.yml

```diff
@@ -0,0 +1,102 @@
+name: macos-arm64
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '**'
+      - '!.github/**'
+      - '!**.yml'
+      - '.github/workflows/macos.yml'
+      - '!**.md'
+      - '!.vscode/**'
+      - '!doc/**'
+      - '!doc/**'
```

Comment:
Duplicated lines. (below too)

Reply:
Fixed. 
I took this from Stratagus, so the lines are duplicated there too.